website/content/docs/connect/gateways/ingress-gateway.mdx

```diff
@@ -68,4 +68,208 @@ must also provide `agent:read` for its node's name in order to discover the agen
 
 ~> [Configuration entries](/docs/agent/config-entries) are global in scope. A configuration entry for a gateway name applies
 across all federated Consul datacenters. If ingress gateways in different Consul datacenters need to route to different
-sets of services within their datacenter then the ingress gateways **must** be registered with different names.
+sets of services within their datacenter, then the ingress gateways **must** be registered with different names.
+
+<!-- Add a "permalink" anchor here since this title is long and may be edited
+     but we need to deep-link to it elsewhere -->
+<a name="sds"></a>
+
+## Custom TLS Certificates via Secret Discovery Service (SDS)
+
+~> **Advanced Topic** This is a low-level feature designed for developers
+building integrations with custom TLS management solutions.
+
+Consul 1.11 added support for Ingress Gateways to serve TLS certificates to
+inbound traffic that are sourced from an external service. The external service
+must implement Envoy's [gRPC Secret Discovery
+Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret)
+(or SDS) API.
+
+The following procedure describes how to configure an ingress gateway with TLS certificates from an SDS source. The instructions assume that you are familiar with Envoy configuration and the SDS protocol.
+
+### 1. Configure Static SDS Cluster(s)
+
+Each Envoy proxy that makes up this Ingress Gateway must define one or more additional [static
+clusters](/docs/connect/proxies/envoy#envoy_extra_static_clusters_json) when registering. These additional clusters define how Envoy should connect to the required SDS service(s). Defining extra clusters in Envoy's bootstrap configuration requires a manual registration of the Ingress Gateway with Consul proxy.
+It's not possible to use the `-register` flag with `consul connect envoy -gateway=ingress` to automatically register the proxy in this case.
+
+The cluster(s) must provide connection information and any necessary
+authentication information such as mTLS credentials.
+
+In this example we will show:
+ - A DNS name to discover the SDS service addresses
+ - Local certificate files for TLS client authentication with the SDS server
+   (the certificates are assumed to be created and managed by some other
+   process)
```

Comment:
```suggestion
The following example will demonstrate how to use:
 - A DNS name to discover the SDS service addresses.
 - Local certificate files for TLS client authentication with the SDS server. The certificates are assumed to be created and managed by some other  process.
```
Our style guide discourages using the royal "we" (dude) unless guidance on a specific topic is coming from HashiCorp.